build.cmd

```diff
@@ -1,11 +1,6 @@
 @echo off
 cls
 
-.paket\paket.bootstrapper.exe
```

Comment:
paket.exe is paket.bootstrapper.exe in disguise ;) That's the magic mode, when you send a command to paket.exe, the bootrapper will download latest Paket version to a temporary location and run it. So basically it's the same as with the bootstrapper (you always have the latest version) while for the user there's only one tiny executable that can be also committed to the repo :+1: